app/components/SearchResults/Dropdown.tsx

```diff
@@ -37,7 +39,9 @@ export const SearchResults = ({results}: {results: SearchResultsProps[]}) => {
         <Link key={i} className="search-result" to={result.url}>
           <Paper />
           <div className="search-result-content">
-            <h5 className="search-result-title">{result.title}</h5>
+            <h5 className={['search-result-title', isMobile ? 'xs-bold' : ''].join(' ')}>
```

Comment:
can you do this with a media query, rather than adding clases?

Reply:
How do I add a class to another class with a media query?

Reply:
you don't :/ This way is fine, but it would be nice avoid explicitly checking the screen size in js, if possible

Reply:
To resolve your first 2 comments, I would need to put 1 js screen size check for the first one.
And for the second one, a check large if loop. 
Should I still resolve the first 2?

Reply:
which comments? I'm not sure I understand

Reply:
The 2 regarding `full-width` class

Reply:
wait, so the items in navBar.css aren't for that component? Why aren't they in the appropriate css file? 
But no point in adding the classes, then :D

Reply:
Those are for the search results. Same-same, maybe slightly different, but same:)